.github/workflows/test-cron.yaml

```diff
@@ -154,20 +156,11 @@ jobs:
       PANTS_REMOTE_CACHE_READ: 'false'
       PANTS_REMOTE_CACHE_WRITE: 'false'
     if: ${{ github.repository_owner == 'pantsbuild' }}
-    name: Bootstrap Pants, test Rust (macOS)
+    name: Bootstrap Pants, test Rust (macOS-x86_64)
     needs: check_labels
-    runs-on: macos-10.15
+    runs-on:
+    - macos-10.15
     steps:
-    - env:
```

Comment:
This was here by mistake. The dependency on `check_labels` has already run this check, we just neglected to remove these steps from this job when we broke that out into its own job.